marlowe-playground-client/src/Simulator/State.purs

```diff
@@ -300,14 +305,19 @@ startSimulation ::
   Term Contract ->
   m Unit
 startSimulation initialSlot contract =
-  updateMarloweState
-    ( {- This code was taken/adapted from the SimulationPage, we should revisit if applyPendingInputs is necesary
+  let
+    initialExecutionState =
+      emptyExecutionStateWithSlot initialSlot contract
+        # over (_SimulationRunning <<< _log) (append [ StartEvent initialSlot ])
```

Comment:
Probably could just add it to `emptyExecutionStateWithSlot`

Reply:
nice one, will change in next PR